lib/insights/events/peerconnection.js

```diff
@@ -68,6 +68,26 @@ class PeerConnectionEvents {
       payload: { peerConnectionId }
     });
   }
+
+  /**
+   * Emit DTLS transport state change event
+   * @param {string} peerConnectionId - Peer connection identifier
+   * @param {('new'|'connecting'|'connected'|'closed'|'failed')} state - DTLS transport state
+   * @returns {void}
+   */
+  dtlsTransportState(peerConnectionId, state) {
+    const levelMap = {
+      failed: 'error',
+      closed: 'warning'
```

Comment:
Closed shouldn't be a warning. We close this manually.

Reply:
Do you prefer to use `debug` for the `closed` state instead? I took the `warning` level from the rtc-spec

Reply:
Yes I think it should be the same level as connecting, connected, etc. No need to change it now if it's already in the spec. We can change it later if you prefer since it requires another deployment right.